docs/tutorial-module/develop.rst

```diff
@@ -193,6 +193,7 @@ the ``page_body``:
       </div>
     {% endblock page_body %}
 
+And finally, the success page which will be rendered after a record is created.
```

Comment:
What about:
And finally, the `success.html` page in `invenio_unicorn/templates/invenio_unicorn/` which will be rendered after a record is created. 

Following the same structure as the previous change with `create.html`.